README.md

```diff
@@ -56,6 +56,32 @@ const YourComponent = () => {
 };
 ```
 
+## Supported color models
+
+The default **react-colorful**'s input/output format is HEX string (like `#ffffff`). In case you need another color model, we provide 5 additional color picker bundles:
```

Comment:
```suggestion
The default **react-colorful**'s input/output format is a HEX string (like `#ffffff`). In case if you need another color model, we provide 5 additional color picker bundles:
```

Reply:
Thanks! Are you sure about `The default ... input/output format is a HEX string`. Maybe we should use `the` here cause "HEX string" is definite thing?